### DISCUSSION TITLE
oauth2 is NOT supported?

### DISCUSSION THREAD

**Discussion by Mkallday** [Budibase/budibase#9037] [Help]
Am I supposed to use OpenID as a workaround for an oauth2 authorization?

**Comment by melohagan** [comment]
Hey @Mkallday 

I'm no expert, but OpenID is an authentication protocol, whereas as OAuth2 is an authorization delegation protocol. 

All of these SSO options use OAuth2: https://docs.budibase.com/docs/authentication-and-sso
